ingress/controllers/nginx/README.md

```diff
@@ -168,7 +165,7 @@ Please check the [tcp services](examples/tcp/README.md) example
 
 Since 1.9.13 NGINX provides [UDP Load Balancing](https://www.nginx.com/blog/announcing-udp-load-balancing/).
 
-Ingress does not support UDP services (yet). For this reason this Ingress controller uses a ConfigMap where the key is the external port to use and the value is
+Ingress does not support UDP services (yet). For this reason this Ingress controller uses a config map where the key is the external port to use and the value is
```

Comment:
Can I hook up an existing http ignress to a config map that exposes a port tcp/udp service? suggest clarifying eitherway